esmvaltool/cmorizers/data/formatters/datasets/merra2.py

```diff
@@ -100,6 +146,16 @@ def _extract_variable(in_files, var, cfg, out_dir):
                 var['short_name'], ', '.join(in_files))
     attributes = deepcopy(cfg['attributes'])
     attributes['mip'] = var['mip']
+    attributes['raw'] = var['raw']
```

Comment:
@glpotter here is how to add attributes to the output netCDF file - they are Global, so you can add whatever you think it's needed. Not a worry if things don't work at first pass, I can come back and fix things :+1: The `attributes` dictionary accepts new members and gets written to output, so just add them in it!